README.md

```diff
@@ -466,6 +466,103 @@ Simbody is installed into the `usr/` directory.  The directory
+
+    $ # CMake configuration
+    $ cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="C:\Program Files\Simbody"
+    $ # Compilation
+    $ make
+    $ # Test
+    $ make test
+    $ # Installation
+    $ make install
+
+Example of instructions where one provides our own Blas and Lapack libraries (to be used in a MSYS terminal):
+
+    $ # CMake configuration
+    $ cmake ../simbody -G"MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX="C:\Program Files\Simbody" \
+      -DCMAKE_C_COMPILER:PATH="C:\Program Files\mingw-w64\i686-5.2.0-posix-sjlj-rt_v4-rev0\mingw32\bin\gcc.exe" \
+      -DCMAKE_CXX_COMPILER:PATH="C:\Program Files\mingw-w64\i686-5.2.0-posix-sjlj-rt_v4-rev0\mingw32\bin\g++.exe" \
+      -DBUILD_USING_OTHER_LAPACK:PATH="C:\Program Files\lapack-3.5.0\bin\liblapack.dll;C:\Program Files\lapack-3.5.0\bin\libblas.dll"
+    $ make
+    $ # Test
+    $ make test
+    $ # Installation
+    $ make install
+
+#### Supported MinGW versions
+
+The table below lists the various versions of MinGW versions supported
+
+OS              | Thread | Exception | Recommended | URL                                                                                                                                                                                                                            |
+----------------|--------|-----------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+32 Bits Version | Posix  | Dwarf     | YES         | [MinGW64 project GCC 5.2.0](http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/5.2.0/threads-posix/dwarf/i686-5.2.0-release-posix-dwarf-rt_v4-rev0.7z/download)    |
+32 Bits Version | Posix  | SJLJ      |             | [MinGW64 project GCC 5.2.0](http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/5.2.0/threads-posix/sjlj/i686-5.2.0-release-posix-sjlj-rt_v4-rev0.7z/download)      |
+64 Bits Version | Posix  | SEH       | YES         | [MinGW64 project GCC 5.2.0](http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/5.2.0/threads-posix/sjlj/x86_64-5.2.0-release-posix-sjlj-rt_v4-rev0.7z/download)    |
+64 Bits Version | Posix  | SJLJ      |             | [MinGW64 project GCC 5.2.0](http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/5.2.0/threads-posix/seh/x86_64-5.2.0-release-posix-seh-rt_v4-rev0.7z/download)      |
+
+For the recommended versions, user does not have to provide its own Blas and Lapack libraries. In the other case, Blas and Lapack have to be compiled manually.
```

Comment:
The table says the recommended version is SEH. But then the user will have to provide their own Blas/Lapack so either we should recommend SJLJ or change this sentence.

It does appear that it would be better if Simbody were using SEH and provided SEH libraries. However, I think that should be the topic of a separate issue now and a later PR.